java/spacewalk-java.spec

```diff
@@ -60,6 +60,12 @@
 %define java_version    1:11
 %endif
 
+%if 0%{?is_opensuse}
+%define supported_locales bn_IN,ca,de,en_US,es,fr,gu,hi,it,ja,ko,pa,pt,pt_BR,ru,ta,zh_CN,zh_TW
+%else
+%define supported_locales en_US,zh_CN
+%endif
```

Comment:
Do we really need to do a replacement in both cases?
For 'master' i would suggest to only replace if **not opensuse**.
The value defined in the config should reflect the Uyuni facts.

Reply:
I have another point of view: this way you don't need to check two files to check what SUSE Manager and what Uyuni has. You can just check the SPEC. And you can even have certain languages for development (on the config file on the sources) and others for releasing.

Reply:
Ok, maybe we need to change the if condition later when Uyuni gets build for oher OSes. But for now it should work

Reply:
> Ok, maybe we need to change the if condition later when Uyuni gets build for oher OSes. But for now it should work

I guess when that happens we'll need to invert this (if SLE do this, else do that). But we'll need to change other conditions, such as the beta tag thing, so I did `if opensuse do this, else do that` to keep it all the same and make things easier to understand now, and to change in the future.